.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -35,17 +36,34 @@ runs:
       run: |
         python -m pip install --upgrade pip
         pip install -r scripts/script-requirements.txt
-    - name: Generate Versioned Documentation
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.1'
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      working-directory: ${{ inputs.working-directory }}
+      run: npm ci
+      shell: bash
+    - name: "Setup Hugo"
+      uses: peaceiris/actions-hugo@v2
+      with:
+        extended: true
+        hugo-version: '0.110.0'
+    - name: "Build"
+      env:
+        KEEP_MASTER: ${{ inputs.keep-master }}
+        THIS_BRANCH: ${{ inputs.this-branch }}
+        BASE_URL: ${{ inputs.base-url }}
+        HUGO_ENV: production
       shell: bash
       run: |
         cd docs
         rm -f generate.sh
         wget https://raw.githubusercontent.com/gooddata/gooddata-python-sdk/master/scripts/generate.sh
         chmod +x ./generate.sh
         ./generate.sh ${{ inputs.fetch-from }} master ${{ inputs.keep-master}}
-    - name: Hugo Build
-      uses: gooddata/gooddata-python-sdk/.github/actions/hugo-build-action@master
-      with:
-        hugo-env: ${{ inputs.hugo-env }}
-        working-directory: ${{ inputs.working-directory }}
-        base-url: ${{ inputs.base-url }}
+        npm install
+        hugo version
```

Comment:
What is the purpose of this command?

Reply:
It is just for having better output, we want to be sure we know which version we are using in case of debugging. I know we state it in the action itself, but I would like to be extra sure. This also shows any minors, that might apply.

Reply:
I think it is not necessary. The version is decided based on
````   
 - name: "Setup Hugo"
      uses: peaceiris/actions-hugo@v2
      with:
        extended: true
        hugo-version: '0.110.0'
```

Reply:
I know it is there, but it does mitigate a problem with debugging the logs. It's in the yaml itself, but this saves the extra step for someone who is not familiar with this workflow.

Reply:
I am still not convinced. It is still redundant. The version can be found in the log from peaceiris/actions-hugo@v2